database/migrations/20241105192209_public-user-id.sql

```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+ALTER TABLE users
+ADD COLUMN uuid UUID DEFAULT gen_random_uuid() NOT NULL;
```

Comment:
Try not to name columns after types.

```suggestion
ADD COLUMN public_id UUID DEFAULT gen_random_uuid() NOT NULL;
```